components/nif/nifkey.hpp

```diff
@@ -18,6 +18,15 @@ struct KeyT {
     float mTension;    // Only for TBC interpolation
     float mBias;       // Only for TBC interpolation
     float mContinuity; // Only for TBC interpolation
+    KeyT() :
+        mValue(T()),
```

Comment:
This will still not work for Ogre::Vector3, because its default constructor does not initialize anything.